Description: Change default for MINphone to 1.
 - Also, bump version to 3.3.2
Origin, upstream: http://makenl.cvs.sourceforge.net/viewvc/makenl/makenl/src/fts5.c?revision=1.7
Origin, vendor: http://sourceforge.net/p/makenl/code, feb5909a96a5a8c6fa1a1f96ce332d4b1f239dd0
Author: Andrew Leary <ajleary@users.sourceforge.net>
Last-Update: 2012-12-29

---
 src/fts5.c    |    4 ++--
 src/version.h |    4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/fts5.c b/src/fts5.c
index 725db1f..a025ce6 100644
--- a/src/fts5.c
+++ b/src/fts5.c
@@ -1,4 +1,4 @@
-/* $Id: fts5.c,v 1.6 2012/11/05 23:54:59 ajleary Exp $ */
+/* $Id: fts5.c,v 1.7 2012/12/29 04:38:59 ajleary Exp $ */
 
 #include <stdio.h>
 #include <string.h>
@@ -69,7 +69,7 @@ char *Levels[] =
   "Pvt", "Point"
 };
 
-int Minphone = 3;
+int Minphone = 1;
 int Alphaphone = 0;
 int Allowunpub = 0;
 int PrivateLevel;
diff --git a/src/version.h b/src/version.h
index 72941b7..e6caf79 100644
--- a/src/version.h
+++ b/src/version.h
@@ -1,8 +1,8 @@
-/* $Id: version.h,v 1.16 2012/12/26 04:21:08 ajleary Exp $  */
+/* $Id: version.h,v 1.17 2012/12/29 04:38:59 ajleary Exp $  */
 
 #include "os.h"
 
-#define MAKENL_VERSION      "3.3.1"
+#define MAKENL_VERSION      "3.3.2"
 
 #define MAKENL_LONG_VERSION "MakeNL " MAKENL_VERSION " (" MAKENL_OS ") compiled with " MAKENL_CC " on " __DATE__ " " __TIME__
 
-- 
1.7.10.4

